src/loader/ident_env.cppo.ml

```diff
@@ -152,6 +155,7 @@ let extract_extended_open o =
       extract_extended_open_items o.open_bound_items
 #endif
 
+let doc_off_mode_on = ref false
```

Comment:
I don't really like this global var - we could instead simply have this as a parameter to `extract_signature_tree_item`

Reply:
Particularly since the scope of the stop comment is only supposed to be until the end of the current module/module type/class/class type (https://caml.inria.fr/pub/docs/manual-ocaml/ocamldoc.html#ss:ocamldoc-stop)